sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ImagePollOptionView.kt

```diff
@@ -0,0 +1,281 @@
+                rightMargin = optionStyle.border.width.dpAsPx(this.resources.displayMetrics)) {
+                addRule(ALIGN_PARENT_BOTTOM)
+            }
+        }
+    }
+
+    private fun bindVoteIndicatorText(optionStyle: ImagePollBlockOptionStyle) {
+        voteIndicatorView.run {
+            text = "\u2022"
+            textSize = optionStyle.font.size * 1.05f
+            setTextColor(optionStyle.color.asAndroidColor())
+            val font = optionStyle.font.weight.mapToFont()
+            typeface = Typeface.create(font.fontFamily, font.fontStyle)
+        }
+    }
+
+    fun initializeOptionViewLayout(optionStyle: ImagePollBlockOptionStyle) {
+        gravity = Gravity.CENTER_VERTICAL
+        alpha = optionStyle.opacity.toFloat()
+        setBackgroundColor(Color.TRANSPARENT)
+
+        val borderRadius = optionStyle.border.radius.dpAsPx(resources.displayMetrics).toFloat()
+        val borderStrokeWidth = optionStyle.border.width.dpAsPx(resources.displayMetrics).toFloat()
+
+        borderPaint = Paint().create(
+            optionStyle.border.color.asAndroidColor(),
+            Paint.Style.STROKE,
+            borderStrokeWidth * 2
+        )
+
+        val rect = RectF(0f, 0f, width.toFloat(), height.toFloat())
+        roundRect = RoundRect(rect, borderRadius, borderStrokeWidth)
+    }
+
+    override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
+        super.onSizeChanged(w, h, oldw, oldh)
+        roundRect = roundRect?.copy(
+            rectF = RectF(
+                0f,
+                0f,
+                width.toFloat(),
+                height.toFloat()
+            )
+        )
+    }
+
+    override fun onDraw(canvas: Canvas) {
+        super.onDraw(canvas)
+
+        roundRect?.let { roundRect ->
+            roundRect.rectF.let {
+                if (roundRect.halfBorderStrokeWidth != 0f) {
+                    canvas.drawRoundRect(
+                        it,
+                        roundRect.borderRadius,
+                        roundRect.borderRadius,
+                        borderPaint)
+                }
+            }
+        }
+    }
+
+    override fun draw(canvas: Canvas?) {
+        roundRect?.let { roundRect ->
+            roundRect.rectF.let {
+                canvas?.clipPath(Path().apply {
+                    addRoundRect(it, roundRect.borderRadius, roundRect.borderRadius, Path.Direction.CW)
+                })
+            }
+        }
+        super.draw(canvas)
+    }
+}
+
+class VotingIndicatorBar(context: Context?): View(context) {
+
+    private val borderPaint = Paint().create(Color.RED, Paint.Style.FILL)
+    private val inset = 4f.dpAsPx(resources.displayMetrics).toFloat()
+    private val rectF by lazy { RectF(inset, 0f, this.width.toFloat() - inset, this.height.toFloat()) }
+
+    override fun onDraw(canvas: Canvas) {
+        super.onDraw(canvas)
+        canvas.drawRoundRect(rectF, 20f, 20f, borderPaint)
```

Comment:
is 20f the corner radius? I feel like this constant also deserves to be factored out